clients/src/main/java/org/apache/kafka/clients/admin/AdminClient.java

```diff
@@ -256,6 +356,21 @@ public CreateAclsResult createAcls(Collection<AclBinding> acls) {
      *
      * This operation is supported by brokers with version 0.11.0.0 or higher.
      *
+     * <p>The following exceptions can be anticipated when calling {@code get()} on the futures obtained from the
+     * {@link CreateAclsResult#values() values()} and {@link CreateAclsResult#all() all()} methods of the  returned
+     * {@code CreateAclsResult}</p>
+     * <ul>
+     *     <li>{@link org.apache.kafka.common.errors.SecurityDisabledException}
+     *     if security features are disabled</li>
+     *     <li>{@link org.apache.kafka.common.errors.ClusterAuthorizationException}
+     *     if cluster authorization failed</li>
+     *     <li>{@link org.apache.kafka.common.errors.InvalidRequestException}
+     *     if invalid resource name was provided</li>
```

Comment:
This is only one reason why the request might be invalid.  Probably best to just say "if the request was invalid" -- the specific exception text explains more about why it was invalid.